File: BEE 1012.cpp
#include<iostream>

#include<iomanip>
#define FIXED_FLOAT(x) std::fixed << std::setprecision(3) << (x)

using namespace std;

int main(){
    float a, b, c;
    cin >> a >> b >> c;
    cout<<"TRIANGULO: "<<FIXED_FLOAT((1.0/2)*a*c)<<endl;
    cout<<"CIRCULO: "<<FIXED_FLOAT(3.14159*(c*c))<<endl;
    cout<<"TRAPEZIO: "<<FIXED_FLOAT((1.0/2)*(a+b)*c)<<endl;
    cout<<"QUADRADO: "<<FIXED_FLOAT(b*b)<<endl;
    cout<<"RETANGULO: "<<FIXED_FLOAT(a*b)<<endl;
    return 0;
}

File: BEE 1012.php
<?php

    list($A, $B, $C) = explode(" ", readline());
    $A = (float)$A;
    $B = (float)$B;
    $C = (float)$C;
    $pi = 3.14159;

    $triangle = (1/2) * $A * $C;
    $triangle = number_format($triangle, 3, ".", "");
    echo "TRIANGULO: $triangle\n";

    $radius_circle = $pi * pow($C, 2);
    $radius_circle = number_format($radius_circle, 3, ".", "");
    echo "CIRCULO: $radius_circle\n";

    $trapezium = (1/2) * ($A + $B) * $C;
    $trapezium = number_format($trapezium, 3, ".", "");
    echo "TRAPEZIO: $trapezium\n";

    $square = pow($B, 2);
    $square = number_format($square, 3, ".", "");
    echo "QUADRADO: $square\n";

    $rectangle = $A * $B;
    $rectangle = number_format($rectangle, 3, ".", "");
    echo "RETANGULO: $rectangle\n";


?>